recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg

```diff
@@ -8,7 +8,7 @@ name = "DomF"
 
 kernel = "/xt/domf/Image"
 
-extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domf ip=192.168.1.13 rw rootwait console=hvc0"
+extra = "root=/dev/mmcblk0p3 rw rootwait console=hvc0"
```

Comment:
don't we need ip=dhcp here?

Reply:
Do we really need dhcp at the kernel boot time? Don't we have a dhcp client in DomF?

Reply:
donno